src/Playwright.TestingHarnessTest/tests/baseTest.ts

```diff
@@ -117,4 +117,14 @@ function unintentFile(content: string): string {
   return lines.join('\n').trim();
 }
 
+// https://github.com/microsoft/vstest/blob/200a783858425e5ac6f4ebb8f87a7811b0ad39e3/src/vstest.console/Internal/ConsoleLogger.cs#L319-L343
+function extractVstestMessages(stdout: string, prefix: string): string {
```

Comment:
nunit/mstest or more in general the visual studio test platform will report the stdout/stderr after each test run with a prefix and two blank lines as a end. This parses them so we can easier process them. This seemed easier to trx, but can be changed most likely to get it from trx instead.